examples/main/constants/toc.js

```diff
@@ -0,0 +1,83 @@
+// Modules
```

Comment:
Nit: `toc.js` is really short. I'd prefer `table-of-contents.js` (and corresponding change in source)